integration_tests/src/main/python/udf_test.py

```diff
@@ -179,11 +173,37 @@ def group_size_udf(to_process: pd.Series) -> int:
 
 low_upper_win = Window.partitionBy('a').orderBy('b').rowsBetween(-3, 3)
 
-udf_windows = [no_part_win, unbounded_win, cur_follow_win, pre_cur_win, low_upper_win]
+running_win_param = pytest.param(pre_cur_win, marks=pytest.mark.xfail(
+    condition=is_databricks_runtime() and is_spark_341(),
+    reason='DB13.3 wrongly uses RunningWindowFunctionExec to evaluate a PythonUDAF'))
+udf_windows = [no_part_win, unbounded_win, cur_follow_win, running_win_param, low_upper_win]
 window_ids = ['No_Partition', 'Unbounded', 'Unbounded_Following', 'Unbounded_Preceding',
               'Lower_Upper']
 
 
+# This test runs only on CPU to verify that DB13.3 wrongly uses RunningWindowFunctionExec
+# to evaluate a PythonUDAF and always fails due to the following error.
+# """
+#  scala.MatchError: pandas_sum(b#1351)#1354L (of class org.apache.spark.sql.catalyst.expressions.PythonUDAF)
+#      at com.databricks.sql.execution.window.RunningWindowFunctionExec.$anonfun$aggregates$2(RunningWindowFunctionExec.scala:117)
+# """
+# (Not sure if it would be fixed in the future releases.)
+@ignore_order
+@pytest.mark.skipif(not (is_databricks_runtime() and is_spark_341()), reason="only for DB13.3")
```

Comment:
Why we only check on Databricks' spark 3.4.1?
I suppose in the later version of Databricks like 13.3+ should also need this case.

Reply:
This is a bug, I think and suppose DB would fix it in the later release.

Reply:
But I am fine to xfail it for later versions. I will file a follow-up issue after merging this.